src/components/index.d.ts

```diff
@@ -2,7 +2,6 @@
 
 import React from 'react'
 declare class UniversalNavbar extends React.Component {
-  private props
```

Comment:
I found some issues unrelated to this PR, but, since I elected to use this opportunity to create an easier way to verify our typings, I think it _sort of_ IS related 😋